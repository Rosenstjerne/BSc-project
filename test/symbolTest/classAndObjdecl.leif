class Point {
    var int x;
    var int y;
}

var Point p;
p = new Point;
p.x = 3;
p.y = 4;

print(p.x); # Should print 3
print(p.y); # Should print 4