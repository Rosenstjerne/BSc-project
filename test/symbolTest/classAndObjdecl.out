function null main() {
	class Point {
		var int x;
		var int y;
		}
	
	var Point p;
	
	p = (new Point);
	(p.x) = 3;
	(p.y) = 4;
	print((p.x));
	print((p.y));
	
}
